governance/votes/Executive Vote - April 29, 2022.md

```diff
@@ -0,0 +1,74 @@
+- A total of 9,758,130 DAI (itemized further down) will be streamed to multiple Core Units.
+
+**Voting for this executive proposal will place your MKR in support of the changes and additions outlined above.**
+
+Unless otherwise noted, the changes and additions listed above are subject to the [GSM Pause Delay](https://manual.makerdao.com/parameter-index/core/param-gsm-pause-delay). This means that if this executive proposal passes, the changes and additions listed above will only become active in the Maker Protocol after the GSM Pause Delay has expired. The GSM Pause Delay is currently set to **48 hours**.
+
+This executive proposal includes an office-hours modifier that means that it **can only be executed between 14:00 and 21:00 UTC, Monday - Friday**. This is to ensure that at least some auction keepers are available to react to unforeseen issues with their bots in the event of early liquidations on new collateral types.
+
+If this executive proposal does not pass within 30 days, then it will expire and can no longer have any effect on the Maker Protocol.
+
+---
+
+## Proposal Details
+
+
+### Core Unit Budget Transfers
+
+As per multiple successful budget proposals (linked below), a total of **1,057,908 DAI** will be transferred to multiple Core Units if this executive proposal passes:
+
+- **800,000 DAI** will be transferred to the [Protocol Engineering Core Unit (PE-001)](https://mips.makerdao.com/mips/details/MIP39c2SP7) at [0xe2c16c308b843eD02B09156388Cb240cEd58C01c](https://etherscan.io/address/0xe2c16c308b843eD02B09156388Cb240cEd58C01c) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmWphKJY#poll-detail).
+- **73,226 DAI** (itemized below) will be transferred to the [Governance Communications Core Unit (COM-001)](https://mips.makerdao.com/mips/details/MIP39c2SP30) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmWQBUi6#poll-detail).
+	- **46,836 DAI** at [0x99E1696A680c0D9f426Be20400E468089E7FDB0f](https://etherscan.io/address/0x99E1696A680c0D9f426Be20400E468089E7FDB0f).
+	- **26,390 DAI** at [0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9](https://etherscan.io/address/0x1eE3ECa7aEF17D1e74eD7C447CcBA61aC76aDbA9).
+- **149,692 DAI** will be transferred to the [Events Core Unit (EVENTS-001)](https://mips.makerdao.com/mips/details/MIP39c2SP32) at [0x3D274fbAc29C92D2F624483495C0113B44dBE7d2](https://etherscan.io/address/0x3D274fbAc29C92D2F624483495C0113B44dBE7d2) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmSVzGcm#poll-detail).
+- **35,000 DAI** will be transferred to the [Strategic Happiness Core Unit (SH-001)](https://mips.makerdao.com/mips/details/MIP39c2SP11) at [0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550](https://etherscan.io/address/0x955993Df48b0458A01cfB5fd7DF5F5DCa6443550) as a result of [their most recent budget approval](https://vote.makerdao.com/polling/QmVdxBE5#poll-detail).
```

Comment:
given that this one is a signal, we should probably call that out and link both the on-chain poll and the signal thread in the forum.